climakitae/core/data_load.py

```diff
@@ -78,8 +80,14 @@ def load(xr_da):
             ),
             end="",
         )
-        print("\r")
-        return xr_da.compute()  # Load data into memory and return
+        if progress_bar:
```

Comment:
Just to keep aligned with Slack: I re-added this ProgressBar since it really helps with visibility into loading from the back-end, but I've added the `progress_bar` condition so that this doesn't affect other ck.load calls. We should definitely look back into the memory spiking issue, but I think for the scope of this CAVA pilot, the ProgressBar does more good than harm for now.